crates/compiler/can/tests/snapshots/test_suffixed__suffixed_tests__expect_then_bang.snap

```diff
@@ -0,0 +1,55 @@
+---
+source: crates/compiler/can/tests/test_suffixed.rs
+expression: snapshot
+---
+main =
+expect Bool.isEq 1 2
+
+x
```

Comment:
Don't understand what happens here. The AST looks correct, probably smth with the formatter

Reply:
To my knowledge, the formatter has never been tested with 'artificial' / transformed ASTs like this - so it's not really surprising that it'd find some bugs.